### PR TITLE
Shamelessly adds Engineering powercreep by stealing valuable original code from Paradise.

### PR DESCRIPTION
**Pull request by lolman360**
## About The Pull Request
Ports a modified version of https://github.com/ParadiseSS13/Paradise/pull/12062 adapted to tgui(now oracleui (god help me).)

## Why It's Good For The Game
Powercreep = good.
Use for autism-level power = good.
Black okayed it.
## Changelog
:cl:
add: Adds a new station goal that generates cool stuff.
/:cl:

### CODE REVIEW

File: code/modules/station_goals/bluespace_tap.dm [hunk 1]
```diff
@@ -0,0 +1,394 @@
...
+	name = "organic objects"
+	loot = list(
+		/obj/item/seeds/random/labelled = 50,
+		/obj/item/guardiancreator/biological = 5,
+		/obj/item/organ/internal/vocal_cords/adamantine = 15,
+		/obj/item/reagent_containers/glass/bottle/reagent/omnizine = 15,
+		/obj/item/dnainjector/xraymut = 5,
+		/obj/item/dnainjector/telemut = 5,
+		/obj/item/dnainjector/midgit = 5,
+		/obj/item/dnainjector/morph = 5,
```
Comment (Trilbyspaceclone) [inline code/modules/station_goals/bluespace_tap.dm]: Why add in self antags

File: code/modules/station_goals/bluespace_tap.dm [hunk 2]
```diff
@@ -0,0 +1,394 @@
...
+	)
+
+/obj/effect/spawner/lootdrop/bluespace_tap/weapon
+	name = "weaponry"
+	lootcount = 1
+	loot = list(
+		/obj/item/gun/energy/e_gun/old = 10,
+		/obj/item/gun/energy/e_gun/cx = 10, //it's a recolourable egun
+		/obj/item/gun/energy/e_gun/turret = 10, //just a shittier HoS egun
+		/obj/item/gun/energy/plasma/MP40k = 5,
```
Comment (Trilbyspaceclone) [inline code/modules/station_goals/bluespace_tap.dm:172]: These are brokenly good

File: code/modules/station_goals/bluespace_tap.dm [hunk 3]
```diff
@@ -0,0 +1,394 @@
...
+		/obj/item/gun/energy/laser/LaserAK = 2,
```
Comment (Trilbyspaceclone) [inline code/modules/station_goals/bluespace_tap.dm:173]: They auto charge

File: code/modules/station_goals/bluespace_tap.dm [hunk 4]
```diff
@@ -0,0 +1,394 @@
...
+			/obj/item/clothing/head/collectable/slime,
+			/obj/item/clothing/head/collectable/police,
+			/obj/item/clothing/head/collectable/slime,
+			/obj/item/clothing/head/collectable/xenom,
+			/obj/item/clothing/head/collectable/petehat
+	)
+
+
+/obj/effect/spawner/lootdrop/bluespace_tap/cultural
+	name = "cultural artifacts"
```
Comment (Trilbyspaceclone) [inline code/modules/station_goals/bluespace_tap.dm:88]: Why is this one the best one?

**Review by Trilbyspaceclone — CHANGES_REQUESTED** [review]
Please remove morph injectors

**Comment by TDSSS** [comment]
Seeing my shitcode ported to another codebase, wew.

**Comment by deathride58** [comment]
wait dont we have oracleui now

**Comment by ghost** [comment]
this sounds awesome

**Comment by Linzolle** [comment]
> 
> 
> wait dont we have oracleui now

oracle isn't merged yet and also the person making this PR would probably use tgui anyway because they said "tgui > oui" on my PR for oracle

**Comment by lolman360** [comment]
i like tgui i just have a reputation for unironically liking it so i cant stop

**Comment by Arturlang** [comment]
Finally a reason for extreme amounts of power, engi mains ahoy!

**Comment by lolman360** [comment]
not stale sorry for not pushing commits but i use webgit

**Comment by silicons** [comment]
stale and a draft too

**Pull request closed without merging**